templates/build_files.go

```diff
@@ -0,0 +1,90 @@
+package templates
+
+var buildFilesTemplate = projectTemplate{
+	Dirs:  getCartridgeBuildDirs(),
+	Files: getCartridgeBuildFiles(),
+}
+
+func getCartridgeBuildDirs() []dirTemplate {
+	return []dirTemplate{}
+}
+
+func getCartridgeBuildFiles() []fileTemplate {
+	preBuildHook := fileTemplate{
+		Path: "cartridge.pre-build",
+		Mode: 0755,
+		Content: `#!/bin/sh
+
+# Simple pre-build script
+# Will be ran before "tarantoolctl rocks make" on application build
+# Could be useful to install non-standart rocks modules
+
+# For example:
+# tarantoolctl rocks make --chdir ./third_party/my-custom-rock-module
+`,
+	}
+
+	postBuildHook := fileTemplate{
+		Path: "cartridge.post-build",
+		Mode: 0755,
+		Content: `#!/bin/sh
+
+# Simple post-build script
+# Will be ran after "tarantoolctl rocks make" on application packing
+# Could be useful to remove some build artifacts from result package
+
+# For example:
+# rm -rf third_party
+# rm -rf node_modules
+# rm -rf doc
+`,
+	}
+
+	buildDockerfile := fileTemplate{
+		Path: "Dockerfile.build.cartridge",
+		Mode: 0644,
+		Content: `# Simple Dockerfile
+# Used by "pack" command as a base for build image
+# when --use-dcoker option is specified
```

Comment:
`dcoker ` -> `docker`.

Also please fix it in master. I think you simply copy-pasted this text

Reply:
I don't want to fix it in a master - this isn't a reason to tag a new `1.8` version.